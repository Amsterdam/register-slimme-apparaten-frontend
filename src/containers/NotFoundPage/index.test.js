/**
 * Testing the NotFoundPage
 */
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { ThemeProvider } from '@amsterdam/asc-ui';

import NotFound from './index';

describe('<NotFound />', () => {
  it('should render the Page Not Found text', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <ThemeProvider>
          <NotFound />
        </ThemeProvider>
      </MemoryRouter>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
