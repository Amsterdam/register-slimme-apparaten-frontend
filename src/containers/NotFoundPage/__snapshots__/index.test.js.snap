// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the Page Not Found text 1`] = `
<DocumentFragment>
  <article>
    <h1>
      Page not found
    </h1>
  </article>
</DocumentFragment>
`;
